ext/snmp/snmp.c

```diff
@@ -862,7 +901,13 @@ static bool netsnmp_session_init(php_snmp_session **session_p, int version, zend
 		host_ptr++;
 		if ((pptr = strchr(host_ptr, ']'))) {
 			if (pptr[1] == ':') {
-				remote_port = atoi(pptr + 2);
+				char *pport = pptr + 2;
+				tmp_port = atoi(pport);
+				if (tmp_port < 0 || tmp_port > USHRT_MAX) {
+					zend_value_error("remote port must be between 0 and %u", USHRT_MAX);
```

Comment:
Use the argument version for it and use the host arg number, as the error message does not point where the error is.